test/integration_test/kubevirt_test.go

```diff
@@ -0,0 +1,77 @@
+//go:build integrationtest
+// +build integrationtest
+
+package integrationtest
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/portworx/sched-ops/k8s/kubevirt"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+)
+
+func TestKubevirt(t *testing.T) {
+	// reset mock time before running any tests
+	err := setMockTime(nil)
+	require.NoError(t, err, "Error resetting mock time")
+
+	t.Run("kubevirtDeployFedoraVMWithClonePVC", kubevirtDeployFedoraVMWithClonePVC)
+}
+
+func kubevirtDeployFedoraVMWithClonePVC(t *testing.T) {
+	var testrailID, testResult = 50803, testResultFail
+	runID := testrailSetupForTest(testrailID, &testResult)
+	defer updateTestRail(&testResult, testrailID, runID)
+	instanceID := "vm"
+	appKey := "kubevirt-fedora"
+	deployedVMName := "test-vm-csi"
+
+	kubevirtVMDeployAndValidate(
+		t,
+		instanceID,
+		appKey,
+		deployedVMName,
+	)
+
+	// If we are here then the test has passed
+	testResult = testResultPass
+	logrus.Infof("Test status at end of %s test: %s", t.Name(), testResult)
+}
+
+func kubevirtVMDeployAndValidate(
+	t *testing.T,
+	instanceID string,
+	appKey string,
+	deployedVMName string,
+) []*scheduler.Context {
+	ctxs, err := schedulerDriver.Schedule(instanceID,
+		scheduler.ScheduleOptions{
+			AppKeys: []string{appKey},
+		})
+	require.NoError(t, err, "Error scheduling task")
+	require.Equal(t, 1, len(ctxs), "Only one task should have started")
+
+	err = schedulerDriver.WaitForRunning(ctxs[0], defaultWaitTimeout, defaultWaitInterval)
+	require.NoError(t, err, "Error waiting for app to get to running state")
+
+	namespace := appKey + "-" + instanceID
+	vms, err := kubevirt.Instance().ListVirtualMachines(namespace)
+	require.NoError(t, err, "Error listing virtual machines")
+
+	for _, vm := range vms.Items {
+		validateVm(t, vm.Name, vm.Namespace, deployedVMName)
+	}
+	return ctxs
+}
+
+func validateVm(t *testing.T, name, namespace, vmName string) {
+	fedoraVm, err := kubevirt.Instance().GetVirtualMachine(name, namespace)
+	fmt.Printf("Fedora VM Name: %s", fedoraVm.Name)
```

Comment:
Where does this message appear? Should we use logrus or t.Logf() instead of fmt.Printf? Also, do you need "\n" in the end?

Reply:
Removed the message.